<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteWidget</class>
 <widget class="QWidget" name="RemoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layout" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="buttons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="buttonRefresh">
            <property name="text">
             <string>&amp;Refresh</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonMkdir">
            <property name="text">
             <string>&amp;New Folder</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonRename">
            <property name="text">
             <string>R&amp;ename</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonPurge">
            <property name="text">
             <string>De&amp;lete</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonMount">
            <property name="text">
             <string>&amp;Mount</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonStream">
            <property name="text">
             <string>&amp;Stream</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonUpload">
            <property name="text">
             <string>&amp;Upload...</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonDownload">
            <property name="text">
             <string>&amp;Download...</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonSize">
            <property name="text">
             <string>&amp;Get Size...</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonExport">
            <property name="text">
             <string>E&amp;xport...</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="path">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="tree">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>true</bool>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="refresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="mount">
   <property name="text">
    <string>&amp;Mount</string>
   </property>
  </action>
  <action name="stream">
   <property name="text">
    <string>&amp;Stream</string>
   </property>
  </action>
  <action name="mkdir">
   <property name="text">
    <string>&amp;New Folder</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="rename">
   <property name="text">
    <string>R&amp;ename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="purge">
   <property name="text">
    <string>De&amp;lete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="upload">
   <property name="text">
    <string>&amp;Upload</string>
   </property>
  </action>
  <action name="download">
   <property name="text">
    <string>&amp;Download</string>
   </property>
  </action>
  <action name="getSize">
   <property name="text">
    <string>&amp;Get Size</string>
   </property>
  </action>
  <action name="export_">
   <property name="text">
    <string>E&amp;xport</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>buttonRefresh</tabstop>
  <tabstop>buttonMkdir</tabstop>
  <tabstop>buttonRename</tabstop>
  <tabstop>buttonPurge</tabstop>
  <tabstop>buttonMount</tabstop>
  <tabstop>buttonStream</tabstop>
  <tabstop>buttonUpload</tabstop>
  <tabstop>buttonDownload</tabstop>
  <tabstop>buttonSize</tabstop>
  <tabstop>path</tabstop>
  <tabstop>tree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
